website_sale_suggest_create_account/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================================
+Suggest to create user account when buying
+==========================================
+
+This module extends the functionality of e-Commerce to suggest
+the buyer to log in or create an account, but without forcing him to do it.
+
+This way you can avoid duplication of partners in your database and offer your
+users a better experience without blocking 1-time buyers.
+
+However, the *Checkout without sign in* button is not highlighted, to
+implicitly encourage users to create the account.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* *Settings > Configuration > General Settings* and enable *Allow external
+  users to sign up*.
+
+Otherwise you will notice no change after installing this module.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Log out.
+* Buy something from your website.
+* Go to your cart.
+
+There you will see the new options.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
```

Comment:
Weird, sometimes he asks for 1 and sometimes for some... :smirk: 

Well, I reviewed commits from #137 here, so that's the consequence. At least do add a description in your pulls for us to know that before losing time on it, please :blush: